apps/api/src/app/topics/topics.controller.ts

```diff
@@ -0,0 +1,46 @@
+import { Body, Controller, Inject, Post, UseGuards } from '@nestjs/common';
+import { ApiExcludeController, ApiOkResponse, ApiTags } from '@nestjs/swagger';
+import { IJwtPayload } from '@novu/shared';
+
+import { CreateTopicRequestDto, CreateTopicResponseDto } from './dtos/create-topic.dto';
+import { CreateTopicCommand, CreateTopicUseCase } from './use-cases';
+
+import { JwtAuthGuard } from '../auth/framework/auth.guard';
+import { ExternalApiAccessible } from '../auth/framework/external-api.decorator';
+import { UserSession } from '../shared/framework/user.decorator';
+import { AnalyticsService } from '../shared/services/analytics/analytics.service';
+import { ANALYTICS_SERVICE } from '../shared/shared.module';
+
+@Controller('/topics')
+@ApiTags('Topics')
+export class TopicsController {
+  constructor(
+    private createTopicUseCase: CreateTopicUseCase,
+    @Inject(ANALYTICS_SERVICE) private analyticsService: AnalyticsService
+  ) {}
+
+  @ExternalApiAccessible()
+  @UseGuards(JwtAuthGuard)
```

Comment:
Maybe better to put the UseGuards on the whole controller instead of the method? So we won't forget it in others ;)